sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -103,6 +104,14 @@ export class Receiver {
    */
   receive(onMessage: OnMessage, onError: OnError, cancellationToken?: Aborter): ReceiveHandler {
     this._throwIfReceiverOrConnectionClosed();
+    this._throwIfAlreadyReceiving();
+    let lastBatchingReceiverSequenceNum: number | undefined;
+    if (this._batchingReceiver) {
+      lastBatchingReceiverSequenceNum = this._batchingReceiver.getSequenceNumber();
+    }
+    if (lastBatchingReceiverSequenceNum && lastBatchingReceiverSequenceNum > -1 && this._receiverOptions) {
```

Comment:
In the constructor, we ensure that `this._receiverOptions` cannot be null or undefined, so why do we need to check it here?